meegkit/utils/trca.py

```diff
@@ -136,3 +136,46 @@ def bandpass(eeg, sfreq, Wp, Ws):
     y = filtfilt(B, A, eeg, axis=0, padtype='odd',
                  padlen=3 * (max(len(B), len(A)) - 1))
     return y
+
+
+def schaefer_strimmer_cov(X):
+    r"""Schaefer-Strimmer covariance estimator.
+
+    Shrinkage estimator using method from [1]:
+    .. math::
+            \hat{\Sigma} = (1 - \gamma)\Sigma_{scm} + \gamma T
+    where :math:`T` is the diagonal target matrix:
```

Comment:
I think we need an empty line between semicolon and the formula for it to render. 

Also, there are a few errors in the docstring: 

```
/home/travis/build/nbara/python-meegkit/meegkit/utils/trca.py:docstring of meegkit.utils.trca.schaefer_strimmer_cov:6: WARNING: Unexpected indentation.

/home/travis/build/nbara/python-meegkit/meegkit/utils/trca.py:docstring of meegkit.utils.trca.schaefer_strimmer_cov:7: WARNING: Literal block ends without a blank line; unexpected unindent.

/home/travis/build/nbara/python-meegkit/meegkit/utils/trca.py:docstring of meegkit.utils.trca.schaefer_strimmer_cov:9: WARNING: Unexpected indentation.

/home/travis/build/nbara/python-meegkit/meegkit/utils/trca.py:docstring of meegkit.utils.trca.schaefer_strimmer_cov:11: WARNING: Literal block ends without a blank line; unexpected unindent.
```

https://travis-ci.com/github/nbara/python-meegkit/builds/223607667#L621